### ISSUE TITLE
Performance loss in NVDA responses

### ISSUE THREAD

**Issue by ollcayy** [nvaccess/nvda#11199]
NVDA does not react as quickly as before. This continues after the 2019.3 update. The problem is not just synthesizer. It continues to respond slowly when using ESpeak.
Recently, NVDA has not come when Windows wakes up.
The problem is common in sound card triggers. I think it's not just the problem. Now the size of the problem is bigger than before 2019.3.
Using Vocalizer, SpeakApi or Espeak does not change anything. He is having this problem in JAWS. When using Vocalizer, only performance is acceptable. NVDA's performance is better than Jaws when SpeakApi is used.
NVDA Performance should now be restored. Right now, jaws stand out unfairly. It should be considered to prevent the sound card from sleeping, to customize the synthesizer drivers, etc. NVDA performance on some of the USB sound cards is pretty good. Moreover, even if the dolbi is active.

**Comment by Adriani90** [comment]
Closing as duplicate of #5172 and related to #10721. Will be fixed by #11024.

**Comment by Brian1Gaff** [comment]
Yes in most newer sound drivers there is a very annoying sleep function you 
cannot turn off.Jaws are now going to include a sound card keep awake 
system, similar to the silencio one which is often used by blind people 
which sends silence to the sound card disguised as sound. grin.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----
